### DISCUSSION TITLE
How do I make connection using SSH tunnel with Attu?

### DISCUSSION THREAD

**Discussion by lamhktommy** [milvus-io/milvus#33039] [Q&A and General discussion]
Is it possible to use Attu to connect to my cloud Milvus instance with the SSH option?

**Comment by shanghaikid** [comment]
what do you mean?

**Reply by lamhktommy** [reply]
Actually I mean proxy connection option using SSH tunnel

**Comment by shanghaikid** [comment]
if you mean ssl, here is an example:  https://github.com/zilliztech/attu?tab=readme-ov-file#attu-ssl-example
